<?xml version="1.0" ?>
<!-- Launch command_joints_node -->
<launch>

    <arg name="package"     default="panda_controllers" />
    
    <!-- flag -->
    <arg name="lab"         value="true"/>
    
    <!-- load -->
    <arg name="yaml_file"   default="$(find panda_controllers)/config/homing.yaml" />
    
    <include file="$(find panda_controllers)/launch/computed_torque_controller.launch" if="$(arg lab)"/>
    
    <group ns="computed_torque_controller">
        <rosparam command="load" file="$(arg yaml_file)" />
        <node name="command_joints_node" pkg="$(arg package)" type="command_joints_node" output="screen"/>
    </group>
    
</launch>